### DISCUSSION TITLE
如何自定义api的提示词？

### DISCUSSION THREAD

**Discussion by yukinochenshangpeng** [immersive-translate/immersive-translate#2111] [Help]
### 概述 | Summary

![Snipaste_2024-09-16_18-20-56](https://github.com/user-attachments/assets/fbb29253-4782-467b-8849-caca7ad5b20a)
![Snipaste_2024-09-16_18-22-09](https://github.com/user-attachments/assets/b977bd2b-c7f0-4bd6-b85a-49c21feb905b)

- chatgpt的api，通用翻译。将原先prompt里的内容删除会没有翻译，但如果不删除原先的内容直接加上我的提示词，则没有用还是原来的正常翻译。而且prompt有3处，不知道在哪里复制粘贴我的提示词。

- 如果可以解决的话，可以将截图给我，
- 下面是我的提示词

我会提供一篇英文或中文文章，请你按照以下要求处理：

1. 对每一句英文或中文进行逐词、逐块翻译。

2. 翻译结果需要按照英文的语序排列，不必调整为符合中文语法的顺序。
3. 如果是英文，则将其逐个词块直接翻译成中文词汇并逐词拼接，形成类似英式中文。
4. 尽量保留句子的复杂结构和长度，不要简化句式。

文章如下：[在此粘贴文章]文章如下：He quickly ran to the store because he realized he had forgotten his wallet.
回复：他 快速 跑 到 商店 因为 他 意识到 他 已经 忘了 他 的 钱包。

**Comment by theowenyoung** [comment]
添加自定义专家之后，选择你自定义的专家即可
